tests/virtualization/yast_virtualization.pm

```diff
@@ -49,9 +50,8 @@ sub run {
     x11_start_program('xterm');
     wait_screen_change { send_key 'alt-f10' };
     become_root;
-    systemctl 'start libvirtd', timeout => 60;
+    manage_libvirt_yast_virtualization;
     wait_screen_change { send_key 'ret' };
-    systemctl 'status libvirtd', timeout => 60;
     # manually start the 'default' network if it is not active
     if (script_run("virsh net-info default |& grep '^Active.*no'") == 0) {
```

Comment:
@Julie-CAO 
I think this would not be required to check `if statement` and record a soft failure, & should be removed. HDYT?

Check here https://openqa.opensuse.org/tests/3675096#step/yast_virtualization/63

it appears that the virsh net-start default command successfully started the 'default' network. So no need to check  now since libvirt is not there.

And also `susetest:/tmp # systemctl --no-pager restart virtnetworkd.service; echo nusX-$7- > /dev/ttySO`
was restarted the daemon in modular libvirt.

I hope this is same that we need.

Reply:
@guoxuguang can you please confirm this part once, thanks

Reply:
LGTM